test/integration/spec/localparticipant.js

```diff
@@ -335,13 +331,16 @@ describe('LocalParticipant', function() {
           });
 
           if (kind === 'data') {
-            // TODO(mroberts): Awaiting a server deploy. Uncomment me once fixed.
-            it.skip('should transmit any data sent through the LocalDataTrack to the Room to each RemoteDataTrack', async () => {
-              const thoseTracks = thoseTracksMap[event];
-              const thoseTracksReceivedData = thoseTracks.map(track => new Promise(resolve => track.once('message', resolve)));
-              dataChannelSendInterval = setInterval(() => thisTrack.send('foo'), 1000);
-              const data = await Promise.all(thoseTracksReceivedData);
-              data.forEach(item => assert.equal(item, 'foo'));
+            ['string', 'arraybuffer'].forEach(dataType => {
+              it(`should transmit any ${dataType} data sent through the LocalDataTrack to the Room to each RemoteDataTrack`, async () => {
+                const data = dataType === 'string' ? 'foo' : new Uint32Array([1, 2, 3]);
+                const thoseTracks = thoseTracksMap[event];
+                const thoseTracksReceivedData = thoseTracks.map(track => new Promise(resolve => track.once('message', resolve)));
+                dataChannelSendInterval = setInterval(() => thisTrack.send(dataType === 'string' ? data : data.buffer), 1000);
+                const receivedData = await Promise.all(thoseTracksReceivedData);
+                clearInterval(dataChannelSendInterval);
```

Comment:
If we moved the `clearInterval` from the `after` block, I don't think we need to define `dataChannelSendInterval` outside this anymore, do we?